### PR TITLE
feat: update java  central repository

### LINKED ISSUE #61: Deploy java module to central repository

### PR DESCRIPTION
**Pull request by dl239**
* **Please check if the PR fulfills these requirements**

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: src/sdk/java/pom.xml [hunk 1]
```diff
@@ -3,15 +3,40 @@
...
+     <licenses>
+        <license>
+            <name>The Apache Software License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+    <developers>
+        <developer>
+            <id>jingchen2222</id>
```
Comment (tobegit3hub) [inline src/sdk/java/pom.xml]: Should you change the developer name? @dl239

**Review by tobegit3hub — APPROVED** [review]
LGTM

File: src/sdk/java/pom.xml (hunk 1 above)
Comment (dl239) [reply src/sdk/java/pom.xml]: yes

File: src/sdk/java/pom.xml [hunk 2]
```diff
@@ -3,15 +3,40 @@
...
+            <url>https://github.com/dl239</url>
+            <organization>4Paradigm</organization>
+            <organizationUrl>https://www.4paradigm.com/</organizationUrl>
+        </developer>
+    </developers>
+    <scm>
+        <connection>scm:git@github.com:4paradigm/HybridSE.git</connection>
+        <developerConnection>scm:git:ssh://git@github.com:4paradigm/HybridSE.git</developerConnection>
+        <tag>HEAD</tag>
+        <url>https://github.com/4paradigm/HybridSE</url>
```
Comment (imotai) [inline src/sdk/java/pom.xml]: why hybridse?

**Review by imotai — APPROVED** [review]
LGTM

**Pull request merged**